src/Command/CommandBusLaunchCommand.php

```diff
@@ -2,44 +2,60 @@
 
 namespace ClearcodeHQ\CommandBusLauncherBundle\Command;
 
+use ClearcodeHQ\CommandBusLauncher\CommandLauncherException;
 use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
 use Symfony\Component\Console\Input\InputArgument;
 use Symfony\Component\Console\Input\InputInterface;
 use Symfony\Component\Console\Output\OutputInterface;
-use ClearcodeHQ\CommandBusLauncher\CommandLauncherException;
 
-class CommandLauncherCommand extends ContainerAwareCommand
+class CommandBusLaunchCommand extends ContainerAwareCommand
 {
     const SUCCESS_CODE = 1;
-    const ERROR_CODE   = 0;
+    const ERROR_CODE = 0;
```

Comment:
Status 0 is usually threat as success exit. http://tldp.org/LDP/abs/html/exit-status.html